docs/ChangeLog.md

```diff
@@ -1,5 +1,21 @@
 # Splunk Operator for Kubernetes Change Log
 
+## 1.0.1 GA(2021-06-01)
```

Comment:
If I am right, this is usually included as part of the "Draft a New Release" functionality, rather than the PR itself - this will allow uploading the yaml files. Am I right @akondur ?

Reply:
It is included in both places. Changelog.md as well as description for pre-release/release